### DISCUSSION TITLE
aotec gen5 controller

### DISCUSSION THREAD

**Discussion by Nifrabar** [zwave-js/zwave-js-ui#3625] [General]
Hi!
Any idea if there are known problems with aotec gen5 stick on raspberry pi 3+?
I am facing pfoblems to start docker with this stick.
I updated the yaml file with the result of the linux comment  ls /dev/serial/by-id but docker fails.  (/dev/serial/by-id/usb-0658_0200-if00)

docker-compose up ends with next error:
Error response from daemon: error gathering device information while adding custom device "/dev/serial/by-id/usb-0658_0200-if00-port0": no such file or directory
Any idea what goes wrong?

Bart

**Comment by robertsLando** [comment]
This `/dev/serial/by-id/usb-0658_0200-if00` is different from `/dev/serial/by-id/usb-0658_0200-if00-port0` (notice the `-port0` suffix. You may just have a typo issue

**Reply by alkempter** [reply]
Hi,
From my knowledge, this device only has an issue with PI4, not with PI3b
my devices statement e.g. looks like:
networks:
      - zwave
 devices:
    - '/dev/serial/by-id/usb-0658_0200-if00:/dev/zwave'
  
Best regards.

**Comment by Nifrabar** [comment]
@ alkempter
Agreed. but to make it working I needed to change inside zwavejs /dev/zwave with an other location. Of course that did not work. But after reverting to /dev/zwave it all worked like a charm.
-Bart
